src/teaching-element/Assessment/MatchingQuestion.vue

```diff
@@ -16,7 +16,7 @@
             v-for="({ dragged, value }, index) in source"
             :key="index"
             class="drag-spot">
-            <span :class="{ dragged }" class="item">{{ value }}</span>
+            <span v-if="!dragged" :class="{ dragged }" class="item">{{ value }}</span>
```

Comment:
If `dragged` is `false` then it is shown but the class is not applied. If `dragged` is `true` it is not shown so the element does not exist. So there is no reason to ever apply the dragged `class`.